cmd/infracost/register.go

```diff
@@ -49,43 +69,40 @@ func registerCmd(ctx *config.RunContext) *cobra.Command {
 
 			fmt.Printf("\nThank you %s!\nYour API key is: %s\n", name, r.APIKey)
 
-			msg := fmt.Sprintf("%s\nYou can now run %s and point to your Terraform directory or JSON/plan file.",
-				fmt.Sprintf("Your API key has been saved to %s", config.CredentialsFilePath()),
-				ui.PrimaryString("infracost breakdown --path=..."),
-			)
-
-			saveAPIKey := true
-
-			if _, ok := ctx.Config.Credentials[ctx.Config.PricingAPIEndpoint]; ok {
-				fmt.Printf("\nYou already have an Infracost API key saved in %s\n", config.CredentialsFilePath())
+			if isRegenerate {
 				confirm, err := promptOverwriteAPIKey()
```

Comment:
```suggestion
                                 fmt.Println()
				confirm, err := promptOverwriteAPIKey()
```

We should add a new line before this prompt

We should print a new line here

Reply:
sure